src/components/searchBar/index.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  useCallback,
+  useState,
+} from 'react';
+
+import { Icon } from '@components/icon';
+import { cn } from '@utils/helpers';
+
+interface SearchBarProps {
+  label: string;
+  classNames?: string;
+  onSearch?: (query: string) => void;
+}
+
+export const SearchBar = (props: SearchBarProps) => {
+  const { label, classNames } = props;
+  const [searchInput, setSearchInput] = useState('');
+
+  const handleSearchInputChange = useCallback((event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
I might think of debouncing the search, if I am not mistaken the search will be executed without the need for hitting the search button?

Reply:
Resolving this as this will be part of another task